apps/dapp/components/home/FeaturedDAOsList.tsx

```diff
@@ -3,17 +3,20 @@
 
 import { PlusIcon, SparklesIcon } from '@heroicons/react/outline'
 import Link from 'next/link'
+import { FC } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { Button, SuspenseLoader } from '@dao-dao/ui'
 
 import { usePinnedDAOs } from '@/hooks'
-import { featuredDaos } from '@/util'
 
 import { FeaturedCard } from '../FeaturedCard'
+import { FeaturedDao } from '../splash'
 import { DaoCardContainer } from './DaoCardContainer'
 
-export const FeaturedDAOsList = () => {
+export const FeaturedDAOsList: FC<{ featuredDaos: FeaturedDao[] }> = ({
+  featuredDaos,
+}) => {
```

Comment:
```suggestion
interface FeaturedDAOsListProps {
  featuredDaos: FeaturedDao[]
}

export const FeaturedDAOsList = ({ featuredDaos }: FeaturedDAOsListProps) => {
```

Using `FC` is now discouraged and will cause breaking changes in future react versions. It no longer implicitly includes `children` but is used that way in most React node until now. The recommended style is just
`export const Component = (props: PropsInterface) => {`